### DISCUSSION TITLE
Confused about paths

### DISCUSSION THREAD

**Discussion by alcaprar** [photoview/photoview#855] [Q&A and Troubleshooting]
Hello,

this is my setup:
- 3 users: `admin`, `ale` and `marti`
- using docker and photos are mounted into `/photos`

This is the current content of the main `/photos` folder. Each of those folders contains other folders (the actual albums).
![immagine](https://github.com/photoview/photoview/assets/11180418/43e84010-725b-4c86-9620-be13a73c9b4f)

This is the users setup:
![immagine](https://github.com/photoview/photoview/assets/11180418/fe59a973-6f72-4a4c-8936-f6d9b053fbc4)

This is what each user sees in the albums page:

## Admin
![immagine](https://github.com/photoview/photoview/assets/11180418/b2e23738-046a-4715-bdc9-87d8af6b6745)

What did I expect? I'd expect to see 4 folders (everything, since it's the _admin_ and it has access to the root `photos` folder): `ale`, `ale-marti`, `marti` and `famiglia-caprarelli.

## Ale

![immagine](https://github.com/photoview/photoview/assets/11180418/194dad22-3d12-4d87-abed-b91bf270eaa5)

What did I expect? I'd expect to see 3 folders: `ale`, `ale-marti` and `famiglia-caprarelli` but it only shows the content of `ale-marti`.

## Marti

![immagine](https://github.com/photoview/photoview/assets/11180418/262eeddf-dd6e-4969-a45e-2b72c07f3b16)

What did I expect? Exactly what is shown. If I open  the two folders I can see the sub folders (the actual albums).

**Comment by alcaprar** [comment]
I think I discovered something odd in the database: `marti` and `ale-marti` had the `parent_album_id` set to `NULL` in the `albums` table. While `ale` and `famiglia-caprarelli` had it set to `1` which is the `id` of the main root folder, `photos`.

Now I tried to things:
1. change it to `1` for the ones that had it `NULL` (`marti` and `ale-marti`), which seemed to me the correct solution, and the `admin` user started to see all the albums, as I expected. While the other 2 users now don't see anything. I suspect it's due to the `onlyRoot: true` in the [grapqhl request](https://github.com/photoview/photoview/blob/228f2cc1e7b2c9079fa81363c49c9c4d60a5cccd/ui/src/Pages/AllAlbumsPage/AlbumsPage.tsx#L9) that make this [query](https://github.com/photoview/photoview/blob/228f2cc1e7b2c9079fa81363c49c9c4d60a5cccd/api/graphql/models/actions/album_actions.go#L25) returns nothing.
2. change it to `NULL` for `ale` and `famiglia-caprarelli`. This resolved the situation for both `ale` and `marti` but made the `admin` a bit weird. It could now see all the 4 folders and also the `photos` one. If opened the `photos` it showed again the 4 folders.

**Comment by MacroPolo** [comment]
Did you Scan each of these users after creation?
